imagebuildah/build.go

```diff
@@ -113,6 +113,8 @@ type BuildOptions struct {
 // Executor is a buildah-based implementation of the imagebuilder.Executor
 // interface.
 type Executor struct {
+	name                           string
```

Comment:
Please add a field for the numeric index of the stage which this `Executor` is responsible for building.  We'll need that information to ensure that `COPY` instructions that try to reference this stage or a later stage, can be rejected.

Reply:
Done, although I think it was rejecting invalid stages before.